lib/cli/src/cli.rs

```diff
@@ -234,9 +234,9 @@ fn wasmer_main_inner() -> Result<(), anyhow::Error> {
         WasmerCLIOptions::Run(Run::from_binfmt_args())
     } else {
         match command.unwrap_or(&"".to_string()).as_ref() {
-            "add" | "cache" | "compile" | "config" | "create-exe" | "help" | "inspect" | "init"
-            | "run" | "self-update" | "validate" | "wast" | "binfmt" | "list" | "login"
-            | "publish" => WasmerCLIOptions::parse(),
+            "add" | "cache" | "compile" | "config" | "create-obj" | "create-exe" | "help"
+            | "inspect" | "init" | "run" | "self-update" | "validate" | "wast" | "binfmt"
+            | "list" | "login" | "publish" => WasmerCLIOptions::parse(),
```

Comment:
I can remove it completely since we parse the `--help` and `-v` before hitting this code path at all.

Done in https://github.com/wasmerio/wasmer/pull/3430/commits/b2d4e975e3ad97040f3c67b62d219e869707fd8f